spec/models/customer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  describe 'relationships' do
+    it { should have_many :customer_items }
+    it { should have_many(:items).through(:customer_items) }
+  end
+
+  it "#add_item" do
```

Comment:
See my notes on creating an `add_item` method. We will want to utilize our join table and associations.